tests/supply_chain/test_supply_chain.py

```diff
@@ -1364,6 +1368,451 @@ def test_consumer_receive_products_after_vlt_days(self) -> None:
         states = consumer_nodes[expected_frame:consumer_node_index:features].flatten().astype(np.int)
         self.assertEqual(required_quantity_2, states[IDX_RECEIVED])
 
+    def test_init_sku_dynamics_OneTimeSkuPriceDemandSampler(self):
```

Comment:
In Python, it is not recommended to use upper case letters in the function name. Just use `test_init_sku_dynamics_one_time_sku_price_demand_sampler`.

Reply:
Or is there any better naming methods?